openshift-container-platform-4/policies/SC-Systems_and_Communications_Protection/component.yaml

```diff
@@ -812,10 +812,11 @@
   implementation_status: planned
   narrative:
     - text: |
-        A complete control response is planned. Engineering progress can be
-        tracked via:
+        At installation time, FIPS mode can be enabled in OpenShift
+        through a flag [1][2]
 
-        https://issues.redhat.com/browse/CMP-420
+        [1] https://docs.openshift.com/container-platform/latest/installing/installing-fips.html
+        [2] https://docs.openshift.com/container-platform/latest/installing/installing_aws/installing-aws-government-region.html#installation-configuration-parameters_installing-aws-government-region
```

Comment:
Nit: why is the AWS doc added explicitly but not azure of GCP?

Reply:
cause I'm lazy. Should I add those links too?

Reply:
done, thanks for the review.